config/prowgen/prowspecs/specs.go

```diff
@@ -108,12 +108,32 @@ var knownBranches map[string]BranchSpec = map[string]BranchSpec{
 
 		primaryKubernetesVersion: "1.30",
 
-		// TODO: test k8s 1.31 here when possible; requires support in the release-1.15 branch on cert-manager
 		otherKubernetesVersions: []string{"1.25", "1.26", "1.27", "1.28", "1.29", "1.31"},
```

Comment:
Comment was out of date. 
1.31 was already added in https://github.com/cert-manager/testing/pull/1060
 * https://github.com/cert-manager/testing/pull/1060